web/Caddyfile

```diff
@@ -0,0 +1,14 @@
+{
+  admin off
+}
+
+:80 {
+	root * /usr/share/caddy
+	file_server
+
+  handle_path /api/* {
+    reverse_proxy http://api:5000 {
```

Comment:
Doesn't it make the docker image depend on the port specified for the `api` service in `docker-compose.yml`?
Can we make it configurable there, and specify the value for the caddy reverse proxy in docker-compose as well?

Reply:
We can remove the proxy, but then the API has to add a CORS policy.

We can use an env variable to define URL with docker compose: https://github.com/tweag/nixpkgs-graph-explorer/pull/52/commits/85edad6bd6ca9f9ac0b39fcc61aa7216011b7f10

Reply:
Looks great now!